analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -150,6 +189,117 @@ class SpdxDocumentFile(
             }
         }
 
+    /**
+     * Maps a given identifier on an spdx package or external document reference
+     * @param doc spdx document containing all packages or external document references the id could refer to
+     * @param source the identifier to be found in the spdx document
```

Comment:
Sorry, I don't exactly know whats wrong here. 
Is it the missing newline between the params and the description?

Because the [KDoc-styleguide](https://kotlinlang.org/docs/kotlin-doc.html#param-name) does it the same way if I am not mistaken.

Reply:
While KDoc supports this JavaDoc-style, we prefer https://kotlinlang.org/docs/kotlin-doc.html#inline-markup.